src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -51,9 +51,6 @@ internal class LaunchSettingsProvider : OnceInitializedOnceDisposed, ILaunchSett
             _launchSettingsFilePath = new AsyncLazy<string>(GetLaunchSettingsFilePathNoCacheAsync, commonProjectServices.ThreadingService.JoinableTaskFactory);
         }
 
-        // TODO: Add error list support. Tracked by https://github.com/dotnet/roslyn-project-system/issues/424
```

Comment:
I've commented on this issue with a link to this commit, so it can be easily reverted when/if that job is ever done.